common/djangoapps/course_modes/views.py

```diff
@@ -185,6 +187,12 @@ def get(self, request, course_id, error=None):
                 context["sku"] = verified_mode.sku
                 context["bulk_sku"] = verified_mode.bulk_sku
 
+        context['currency_data'] = []
+        if waffle.switch_is_active('local_currency'):
+            if 'edx-price-l10n' not in request.COOKIES:
+                currency_data = get_currency()
+                context['currency_data'] = currency_data
```

Comment:
Nit: `context['currency_data'] = get_currency()` and avoid the variable assignment?

Reply:
changed